app/models/hesa/subject.rb

```diff
@@ -0,0 +1,23 @@
+# Based on: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/lib/hesa/subject.rb
+module Hesa
```

Comment:
given that this class is essentially just wrapping the subjects list from the reference data gem, i'd argue against schools experience "knowing" anything about hesa.

Reply:
thinking about it, i'd err on the side of getting rid of this class and allowing the calling code (`available_degree_subjects`) to go direct to `DfE::ReferenceData::Degrees::SUBJECTS.all.map(&:to_h)`.

not convinced there's a need to wrap what comes back from the reference data gem. 

what you think?